fdbclient/BlobCipher.cpp

```diff
@@ -560,7 +598,9 @@ Reference<BlobCipherKey> BlobCipherKeyIdCache::insertBaseCipherKey(const Encrypt
 
 	Reference<BlobCipherKey> cipherKey =
 	    makeReference<BlobCipherKey>(domainId, baseCipherId, baseCipher, baseCipherLen, salt, refreshAt, expireAt);
-	keyIdCache.emplace(cacheKey, cipherKey);
+	auto result = keyIdCache.emplace(cacheKey, cipherKey);
```

Comment:
Do we need to check `itr->second->isValid()` here: https://github.com/apple/foundationdb/pull/9845/files#diff-74240a17fd3787bf5b7b99522cd19290378f76c7f14efaf29be9ab8699dbe92cR575?

Reply:
Not sure if that gives us much, we are constructing the BlobCipherKey above, hence, would be valid?

Reply:
Oh i missed that makes sense